packages/apollo-collaboration-server/src/changes/changes.service.ts

```diff
@@ -105,11 +114,164 @@ export class ChangesService {
           `Error in backend post-validation: ${errorMessage}`,
         )
       }
+      this.logger.debug(`TypeName: ${change.typeName}`)
+
+      // Assembly related changes are broadcasted to 'COMMON' -channel
+      if (
+        change.typeName !== 'AddAssemblyAndFeaturesFromFileChange' &&
+        change.typeName !== 'AddAssemblyFromFileChange' &&
+        change.typeName !== 'DeleteAssemblyChange'
```

Comment:
Instead of checking type names explicitly, this should check based on the Change class hierarchy. For example, if a `Change` is an `AssemblySpecificChange` but not a `FeatureChange`, then it should be broadcast on the common channel.

Reply:
Done